src/UnityPackages/io.chainsafe.web3-unity/Runtime/Scripts/ProjectConfigUtilities.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public static class ProjectConfigUtilities
+{
+    public static ProjectConfigScriptableObject Load()
+    {
+        var projectConfig = Resources.Load<ProjectConfigScriptableObject>("ProjectConfigData");
+        return projectConfig ? projectConfig : null;
+    }
+
+#if UNITY_EDITOR
```

Comment:
Oh snap, I've added some similar logic at #453. We'd have to remove the duplicating logic from my side.

Reply:
yep.